### DISCUSSION TITLE
How to tell which sentence is the key in a paragraph?

### DISCUSSION THREAD

**Discussion by bounaux** [explosion/spaCy#12339] [Help: Coding & Implementations] [answered]
I have a list of key words [k1, k2, k3, k4]. 

I have some paragraphs that I'm looking for key words counts, i.e.:

p1 has k1 occurance x count, k2 occurance y, and k3 occurance z. 

If p1 contains 10 sentences, sentence 8 and 9 contain most of the key words occurance, then do I have to run iteration over sentences? or if spacy can return a "socred" sentence list of "relevancy"?

**Comment by adrianeboyd** [comment, marked as answer]
spacy can provide the general-purpose building blocks here (finding keywords with the `Matcher` or `PhraseMatcher`, finding sentences with `sentencizer` or `senter` or `parser`), but it doesn't come with any specific methods like this. You'd need to implement your own method for the specific relevancy measure that you need.

**Reply by bounaux** [reply]
ok understand, thank you!
